packages/react-frontend/src/routes/MyReviews.jsx

```diff
@@ -1,11 +1,87 @@
 import Navbar from "./Navbar";
+import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import Review from "../../components/Review";
+import "./MyReview.css";
+
 export default function MyReviews() {
+  const { username } = useParams();
+  const [user, setUser] = useState(null);
+  const [reviews, setReviews] = useState(null);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        let response = await fetch(
+          `http://localhost:8000/reviews/${username}`
+        );
+        if (!response.ok) {
+          throw new Error(
+            `Network response was not ok: ${response.statusText}`
+          );
+        }
+        let data = await response.json();
+        setUser(data.user);
+        setReviews(data.reviews);
+      } catch {
+        console.error("Error fetching reviews");
+      }
+    }
+    fetchData();
+    console.log(user, reviews);
+  }, []);
+
+  async function handleDelete(reviewId) {
+    console.log(reviewId);
+    try {
+      const response = await fetch(
+        `http://localhost:8000/reviews/user/${reviewId}`,
+        {
+          method: "DELETE",
+          headers: {
```

Comment:
For the purpose of route protection, we will need to add the authorization header to the headers JSON field for all of our protected routes.  Logic looks solid tho!
```suggestion
      headers: {
       "Content-Type": "application/json",
       "Authorization": "Bearer ${token}"
       }
```

Reply:
Thanks. I'll make sure to add it to my code